src/main/resources/db/migration/common/V1_206__update_media_request_expiry_ts.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE media_request ADD expiry_ts TIMESTAMP WITH TIME ZONE;
+ALTER TABLE media_request ADD created_by INTEGER NOT NULL;
```

Comment:
these columns are being added by my PR, which includes all changes in V42 and v43 SQL.Mine should go first, as it includes all changes in those scripts